app/rasabot.py

```diff
@@ -37,18 +38,19 @@ def trainning(self, language, data):
         trainer = Trainer(RasaNLUConfig(config))
         trainer.train(training_data)
         bot_data = trainer.persist()
-        bot = Bot.create(bot=bot_data)
-        bot.save()
-        if bot.uuid:
-            return dict(uuid=str(bot.uuid))
-        else:
-            print("Fail when try insert new bot")
+        with DATABASE.execution_context():
+            bot = Bot.create(bot=bot_data)
+            bot.save()
+            if bot.uuid:
+                return dict(uuid=str(bot.uuid))
+            else:
+                print("Fail when try insert new bot")
```

Comment:
`logging` lib - https://docs.python.org/2/library/logging.html - here would be better instead of `print` command.